doc.go

```diff
@@ -32,6 +32,8 @@
 // In this case, the string "sha256" is the algorithm and the hex bytes are
 // the "digest".
 //
+// This package exclusively uses hex encoding for all digests.
```

Comment:
... for currently supported algorithms. The hex limitation is about the encoding of digest that is bound to the algorithm. New "algorithms" may support other kinds of encodings.